Altis_Life.Altis/core/functions/TON/cell/fn_emsrequest.sqf

```diff
@@ -0,0 +1,17 @@
+/*
+    File: fn_emsrequest.sqf
+    Author: Bryan "Tonic" Boardwine
+
+    Description: send message to ems
+*/
+ctrlShow[3022,false];
+private _msg = ctrlText 3003;
+private _length = count (toArray(_msg));
+if (_length > 400) exitWith {hint localize "STR_CELLMSG_LIMITEXCEEDED";ctrlShow[3022,true];};
```

Comment:
We can count string directly rather then turn to an array

Reply:
why exactly do we check for 400 characters here? I get it, that long messages could be displayed improperly. but why 400? is it arbitrary? is it tested? and why isnt that a config_master setting :D

Reply:
Sounds like a good idea :)

Reply:
done and implemented :D